target/linux/ath79/dts/qca9533_tplink_cpe220-v3.dts

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca9533_tplink_cpexxx.dtsi"
+
+/ {
+	compatible = "tplink,cpe220-v3", "qca,qca9533"; 
+	model = "TP-Link CPE220 v3"; 
+
+	leds {
```

Comment:
See my leds comment above. Same applies here (and you can also remove the gpio-leds, as it will be inherited from parent).